README.md

```diff
@@ -11,6 +11,17 @@ check out our [Getting started page](http://go.microsoft.com/fwlink/?LinkID=7983
 
 Also, don't forget to check out [the documentation](https://aka.ms/dotnet-cli-docs). 
 
+Changes in issue triaging
+-------------------------
+As it was explained in the ["Changes to project.json" blog post](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/), we are making significant changes to the CLI tooling by supporting MSBuild as the build engine and `csproj` as the project format. As we are starting that work, we've realized that this also means that many of the shortcomings of the current tooling will be solved by this move. Given this, we've decided to introduce two new labels for issues to indicate that. These labels are: 
+
+* `msbuild-mitigated` label - this label indicates that we belive the issue in question will be mitigated via move to MSBuild. 
+* `msbuild-notapplicable` label - this label indicates that we believe the issue in question is will not be applicable after the move to MSBuild. 
+
+We will be triaging and applying these labels to issues in the coming days and we will be closing issues we label. This is done so we can decrease the number of overall issues and move the focus towards those issues that we need to fix in the CLI "proper", so to speak. 
```

Comment:
I don't many people will understand your meaning of "CLI 'proper'" here. Doesn't feel like a useful distinction. I think you could just say:

> coming days

Statements like that go stale almost immediately. You should phrase it more as:

> We close issues that are labelled with one of these labels. Proactive triaging will help us reduce the number of issues and enable us to focus on issues that require fixing  independent of the move to msbuild.